src/shared/locales/ptBr/translations/incidents/index.ts

```diff
@@ -0,0 +1,44 @@
+export default {
+  incidents: {
+    filterTitle: 'Filtrar por status',
+    label: 'Incidentes',
+    actions: {
+      report: 'Relatório',
+    },
+    status: {
+      reported: 'Criado',
+      resolved: 'Resolvido',
+      all: 'All',
```

Comment:
By no means I'm an expert on PR-BR but several of those tags seems to be in English 🤔

Reply:
All was my bad! I'll fix it